backend/src/core/routes/campaign.routes.ts

```diff
@@ -9,6 +9,11 @@ const listCampaignsValidator = {
   [Segments.QUERY]: Joi.object({
     limit: Joi.number().integer().min(1).max(100).default(10),
     offset: Joi.number().integer().min(0).default(0),
+    type: Joi.string().valid(...Object.values(ChannelType)),
```

Comment:
NB called "campaign type"

Reply:
on the frontend, this same field is called "Mode" 🙄️
 
<img width="908" alt="image" src="https://user-images.githubusercontent.com/67887489/184289364-318a8783-1c0d-4bcc-bfab-fd10666011b3.png">

Reply:
Standardizaton in separate PR